_3_design_patterns/src/main/java/code/_3_in_class/Main.java

```diff
@@ -1,8 +1,9 @@
 package code._3_in_class;
 
-public class Main {
+public class HelloWorld {
```

Comment:
Este o regula in Java care spune ca numele clasei publice trebuie sa fie identic cu numele fisierului.
In cazul tau fisierul se numeste "Main.java" iar clasa publica "HelloWorld". Fisierul si clasa publica trebuisa aiba acelasi nume.